templates-cache/src/main/scala/activator/templates/TemplateCacheSeedGenerator.scala

```diff
@@ -52,9 +53,22 @@ object TemplateCacheSeedGenerator {
           location = cacheDir,
           remote = remoteRepo,
           autoUpdate = false)
+
+      val templateList =
+        if (arg.templates.isEmpty) cache.featured
+        else {
+          cache.metadata.map { it =>
+            val filtered = it.filter { tpl =>
+              arg.templates.contains(tpl.name)
+            }
+            if (filtered.toList.length == 0) throw new RuntimeException(s"Could not find any template matching name(s): ${arg.templates.mkString(",")}")
```

Comment:
isEmpty is better practice for zero-length, but maybe even better here would be `arg.templates.length == filtered.length` ?  otherwise you don't catch if one name is bad